README.md

```diff
@@ -61,16 +61,25 @@ in your account, note down the "Internal Integration Token" and [link the
 integration][res-notion-link-integration] to the to the duplicated page in your
 account.
 
+For Google Analytics support, you will also need to create a Google Analytics
+account, property and data stream, and note down the measurement ("G-") ID as
+described [here][res-google-analytics-docs] (for setting up data collection,
+follow the instructions to 'Find your "G-" ID' at the bottom of the page).
+
 Create a file `.env.local` in the repository's root directory with the
-following command (don't forget to replace `YOUR_TOKEN` with your internal
-integration token):
+following command (don't forget to replace the values with your own ones):
```

Comment:
This can be marked optional!

Reply:
But is it build in a way that it's optional? Nothing will break if it's not set?

Reply:
It is not set right now. Tanya PR is yet to be merged. We should make it optional as I think have google analytics setup is truly a hassle & is not needed for the developed working on project.

Reply:
Ah okay, I misunderstood it. Will remove all mentions from Google Analytics until it is merged then.